File: UserApiService_C#/AdminApiService/Models/Requests/UserAskPriceRequest.cs
using Newtonsoft.Json;

namespace AdminApiService.Models.Requests;

public class UserAskPriceRequest
{
    [JsonProperty("userCategoryIds")]
    public List<long> userCategoryIds { get; set; }
    [JsonProperty("categoryId")]
    public long categoryId { get; set; }
    [JsonProperty("locationId")]
    public long locationId { get; set; }
    [JsonProperty("date")]
    public long date { get; set; } 
}

File: AvitoStatistcService_Java/src/main/java/ru/espada/avito/statistic/model/request/UserAskPriceRequest.java
package ru.espada.avito.statistic.model.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.ArrayList;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
// Зпрос для изменеия статистики пользователей (после того как пользователь запросил цену)
public class UserAskPriceRequest {

    private ArrayList<Long> userCategoryIds;
    private long categoryId;
    private long locationId;
    private long date;

}
